lib/automation/report_runner.rb

```diff
@@ -70,6 +70,14 @@ def initialize(
 
       def run!
         start_date = (@offset + @days).days.ago
+
+        @title =
+          @title.gsub(
```

Comment:
```suggestion
          @title.gsub!("%DATE%", "#{start_date.strftime("%Y-%m-%d")} - #{end_date.strftime("%Y-%m-%d")}"
```

Does the gsub in place with gsub!, and makes TGX happy because he prefers string interpolation :)

Reply:
yeah but that amend the caller... I dislike there

Reply:
Ah yep fair enough :+1: